velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -23,13 +23,23 @@
 #include "velox/functions/prestosql/registration/RegistrationFunctions.h"
 #include "velox/parse/TypeResolver.h"
 
+#include <map>
+#include <ostream>
+#include <vector>
+
 using namespace facebook::velox;
 using namespace facebook::velox::exec;
 using namespace facebook::velox::exec::test;
 
 static const int kNumDrivers = 4;
 
-class ParquetTpchTest : public testing::Test {
+struct QueryParams {
+  int queryId;
```

Comment:
Prefix "query" is redundant as all these variables are scoped to QueryParams struct.